src/webapi/Controllers/EvaluationController.cs

```diff
@@ -98,52 +94,39 @@ public async Task<ActionResult<List<AvailableEvaluationObjective>>> GetEvaluatio
     [HttpGet("Sync")]
     public async Task<ActionResult> SyncEvaluationComponents()
     {
-        try
```

Comment:
This diff looks bigger than it really is due to the change in indent when removing the `try ... catch`.